emsdk.py

```diff
@@ -665,6 +665,11 @@ def download_file(url, dstpath, download_even_if_exists=False, filename_prefix='
         sys.stdout.flush()
   except Exception as e:
     print("Error downloading URL '" + url + "': " + str(e))
+    # Modified code to inform solution for common "...[SSL: CERTIFICATE_VERIFY_FAILED] ..." on Mac and
+    # "<urlopen error unknown url type: https>" on Linux
+    # Error results in issues raised for similar problem: #133 #136 #140 #176 #6275 #6548 #6723 #9036  ...
+    if "SSL: CERTIFICATE_VERIFY_FAILED" or "urlopen error unknown url type: https" in str(e):
```

Comment:
This is wrong, I think: it will be interpreted as `if "a" or ("b" in c)`. I think we need `if "a" in c or "b" in c`.

Reply:
Oh my, of course. Haven't resolved the Linux version, so might also change the link once I have it worked out on Linux.